app/containers/App/reducer.js

```diff
@@ -41,9 +44,7 @@ function appReducer(state = initialState, action) {
         .set('loading', false)
         .set('currentUser', action.username);
     case LOAD_REPOS_ERROR:
-      return state
-        .set('error', action.error)
-        .set('loading', false);
+      return state.set('error', action.error).set('loading', false);
```

Comment:
I think that state props should not use multiple datatypes. In this reducer `error` is sometimes set to `true` and sometimes set to the `action.error`

Should we store the result of the `ErrorBoundary` in the redux state at all? Is it within the scope of pushing through a React 16 upgrade?